### PR TITLE
Youtube trial branch for pull request

### PR DESCRIPTION
**Pull request by Habtamu210**
This project includes:
-HTML code for you tube
- CSS files for styling you tube trial

### CODE REVIEW

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Habtamu210, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights
- No linter errors  ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

# Please 🙏🏽 do not merge a branch that haven't been approved yet.

- Play the role of the Code Reviewer:
    - Follow the list from the article ["How to give a code review?"](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md).
    - Check if you have used semantic HTML tags (see examples of these tags in the first bullet of the best practices document below).
    - Check if you applied [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
- Add as many comments as possible in the given time and finish your review. Remember to add a minimum of 2 comments - you will definitely find something that could have been done better!

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: youtube.html [hunk 1]
```diff
@@ -9,14 +9,13 @@
...
 <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
  
-
 <link rel="stylesheet" href="style/video.css">
 <link rel="stylesheet" href="style/header.css">
 <link rel="stylesheet" href="style/general.css">
 <link rel="stylesheet" href="style/Sidebar.css">
 </head>
 <body>
-  <div class="header">
```
Comment (Habtamu210) [inline youtube.html:19]: Kindly, use semantic tags instead of generic div tag.

File: youtube.html [hunk 2]
```diff
@@ -9,14 +9,13 @@
 <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
 <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
  
-
```
Comment (Habtamu210) [inline youtube.html:12]: Please remove extra space as that helps to make code more readable

File: youtube.html [hunk 3]
```diff
@@ -9,14 +9,13 @@
...
 <link rel="stylesheet" href="style/header.css">
 <link rel="stylesheet" href="style/general.css">
 <link rel="stylesheet" href="style/Sidebar.css">
 </head>
 <body>
-  <div class="header">
+  <section class="header">
     <div class="left-section">
       <img class="hamburger-menu"src="icon/humberger.png">
       <img class="youtube-menu"src="icon/you.png">
```
Comment (Habtamu210) [inline youtube.html:21]: Kindly, use semantic tags

File: youtube.html [hunk 4]
```diff
@@ -219,6 +218,6 @@
             </div> 
           </div>
           </div>
-  </div>
+        </section>
```
Comment (Habtamu210) [inline youtube.html:221]: Kindly, improve the formatting of the elements to improve readability

File: youtube.html [hunk 5]
```diff
@@ -219,6 +218,6 @@
...
  </body>
 </html>
```
Comment (Habtamu210) [inline youtube.html:223]: Kindly, remove unnecessary blank new line

**Review by Habtamu210 — COMMENTED** [review]
Good job so far 👍 

Please consider the comments given below to improve the site

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @Habtamu210 👋

Your project is complete! ✔️ 🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

### Optional changes

[ ] Consider using a couple of emojis and/or gifs in the review. They make the code review more 
human-friendly 👽🤖.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by PraveenKumar141095** [comment]
Hi @Habtamu210  

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congratulations


## Good Highlights✔

- Good commit messages
- Implemented requested changes
- Adherence to GitHub flow

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**